xoa_driver/functions/anlt.py

```diff
@@ -69,22 +85,42 @@ def obtain_ports_of_module(
     return ports
 
 
-def obtain_port(
-    tester: GenericAnyTester,
-    module_id: int,
-    port_id: int,
-) -> GenericAnyPort:
+def obtain_ports(
+    tester: GenericAnyTester, module_id: int = -1, port_id: int = -1
```

Comment:
`-1` is an misleading value from the user perspective, coz on tester we are not having slot with the idx `-1` the same is relevant to the portbetter to define multiple function annotations by using `@overload` decorator